src/server/auth/server/testing/auth_test.go

```diff
@@ -1103,6 +1103,10 @@ func TestListAndInspectRepo(t *testing.T) {
 	require.Equal(t,
 		buildBindings(alice, auth.RepoOwnerRole), getRepoRoleBinding(t, aliceClient, repoNone))
 
+	// put a file in the repo Bob can't access - we need to be able to get the size of the commits
+	err := aliceClient.PutFile(client.NewCommit(repoNone, "master", ""), "/test", strings.NewReader("test"))
```

Comment:
should we assert that there's a certain amount of bytes in the repo?

Reply:
I'm not too concerned about asserting that here, PFS has it's own test that the results are accurate, here we're just testing the permissions model.